### PR TITLE
Fix read style for webhosting pipeline

### PR DESCRIPTION
**Pull request by TylerZeroMaster**
This change should allow webhosting pipeline to advance to the point where it will check the books.xml for the style.

### CODE REVIEW

File: dockerfiles/docker-entrypoint.sh [hunk 1]
```diff
@@ -101,18 +101,21 @@ function do_xhtml_validate() {
 
 function read_style() {
     slug_name=$1
-    if [[ ! -f $IO_BOOK/style ]]; then
-        die "Style not specified. File does not exist '$IO_BOOK/style'" # LCOV_EXCL_LINE
-    else
+    style_name=
```
Comment (m1yag1) [inline dockerfiles/docker-entrypoint.sh:104]: Is the style_name ok being defined w/o a value? Perhaps, it should default to an empty string?

**Review by m1yag1 — APPROVED** [review]

File: dockerfiles/docker-entrypoint.sh (hunk 1 above)
Comment (TylerZeroMaster) [reply dockerfiles/docker-entrypoint.sh:104]: In bash scripts, that is the equivalent of initializing a variable as None/null. I could change it to an empty string if that's preferable. It should not impact functionality: empty strings are falsey.

Comment (m1yag1) [reply dockerfiles/docker-entrypoint.sh:104]: Well, I'm not sure what's best practice in bash. My personal preference is to set variables to None or an empty string if they need to be empty initially.

Comment (m1yag1) [reply dockerfiles/docker-entrypoint.sh:104]: If that's what Phil has done in previous scripts feel free to keep it the same.

**Pull request merged**